packages/server/src/fhir/operations/utils/parameters.ts

```diff
@@ -223,13 +187,8 @@ export async function sendOutputParameters(
     }
   }
 
-  try {
-    validateResource(response);
-    res.status(getStatus(outcome)).json(response);
-  } catch (err: any) {
-    getLogger().error('Malformed operation output Parameters', { error: err.toString() });
```

Comment:
Up to you whether we want to keep this log; it was intended to help catch server issues